importers/wikidata.py

```diff
@@ -37,19 +38,27 @@ def load_person_from_wikidata(wikidata_url):
     else:
         return {}
 
-
 @cache.dict()
 def load_person_from_wikipedia(wikidata_url, language):
     """Given a wikidata url, get information from wikipedia
     TODO: Allow a wikipedia URL as argument too
     TODO: Check language against valid list
     TODO: Image from wikipedia is different to that of wikidata"""
-    entity = get_entity_for_wikidata(wikidata_url)
+    
+    if 'wikipedia' not in wikidata_url:
+        entity = get_entity_for_wikidata(wikidata_url)
+        wikipedia_url = get_url_for_wikipedia(entity, language)
+        description = get_description_for_wikipedia(entity, language)
+        label = entity.label.get(language)
+        # TODO: Remove html tags from the description
+    else:
+        wikipedia_url = wikidata_url
+        parts = urlparse(wikipedia_url)
+        wd_name = parts.path.split("/")[-1].replace("_", " ")
+        pg = wikipedia.page(wd_name)
+        label = pg.title
+        description = pg.summary
```

Comment:
Is there a reason why `get_description_from_wikipedia` doesn't work here? I think we should only add another dependency if the reason for doing so is clear. I think we have enough methods in this file already to do the task that we need.

Reply:
The reason is that for retrieving data from wikidata, the wikidata client needs as input the wikidata ID 
https://github.com/dahlia/wikidata/blob/main/wikidata/client.py#L130

This was the reason why the method failed and returned None in the case of passing as input the wikipedia URL. In the wikidata module I do not find a "search" method for querying a string and retrieving the wikidata ID, so I used directly the wikipedia module instead